### ISSUE TITLE
xExtension-YouTube warnings in stream

### ISSUE THREAD

**Issue by pax0707** [FreshRSS/Extensions#222]
This happens when enabling "Display the feeds content" in extension options with latest extension and edge build.
Wasn't happening with the build few days ago when I was testing.

```
Warning: DOMDocument::loadHTML(): Tag figure invalid in Entity, line: 2 in /var/www/FreshRSS/extensions/xExtension-YouTube/extension.php on line 203

Warning: DOMDocument::loadHTML(): Tag figcaption invalid in Entity, line: 2 in /var/www/FreshRSS/extensions/xExtension-YouTube/extension.php on line 203
```

![image](https://github.com/FreshRSS/Extensions/assets/3536230/2505a34c-203d-4381-9b4d-63e093a4ae42)

**Comment by karmicdude** [comment]
+1

**Comment by Alkarex** [comment]
Is it still a problem?

**Comment by pax0707** [comment]
@Alkarex yes @ 1.25.1-dev.



![Image](https://github.com/user-attachments/assets/ee22e596-def3-4dae-8ef6-3672879d1195)

**Comment by Alkarex** [comment]
Ping @kevinpapst or @ImAReplicant for a little help if you are around

**Comment by kevinpapst** [comment]
Yeah, I think Youtube change the embedded HTML and the PHP Doc Parser does not support HTML5 tags like figure. 

That's what 8.4 brings in: https://www.php.net/releases/8.4/en.php#dom_additions_html5

Not sure what to do. Shall we add the @ shut up operator?

**Comment by Frenzie** [comment]
> That's what 8.4 brings in

That sounds like the solution for the image proxy as well. But what to do in the meantime. :)

**Comment by Alkarex** [comment]
Related to https://github.com/FreshRSS/FreshRSS/issues/6831
I think the approach is to start using the new functions and define a fallback compatible with PHP 8.1 (using the old functions and hiding the errors at least). I will try to draft something in the core. In the meantime, hiding the errors would be good
